ext/fs/ops.rs

```diff
@@ -382,6 +382,30 @@ where
   Ok(())
 }
 
+#[op2(fast, stack_trace)]
+pub fn op_fs_fchmod_sync(
+  state: &mut OpState,
+  #[smi] rid: ResourceId,
+  mode: u32,
+) -> Result<(), FsOpsError> {
+  let file =
+    FileResource::get_file(state, rid).map_err(FsOpsErrorKind::Resource)?;
```

Comment:
This is not really correct - `ResourceId` is not a real file descriptor that the Node API expects. That said, it appears that we already have several APIs that do that already, so I think it's fine for now...